cmd/package.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+
+	pkgserverv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/package-server/apis/operators/v1"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+func errorPreRunE(message string, err error) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		if err != nil {
+			return fmt.Errorf("%s: %v", message, err)
+		}
+		return fmt.Errorf("%s", message)
+	}
+}
+
+func packageCmd() *cobra.Command {
+	// Run is empty. Otherwise, on an error, it would not be marked
+	// as Runnable, which would not print out the usage/help.
+	// TODO: Can we add the subcommand before the client is established?
+	cmd := cobra.Command{
+		Use:   "package",
+		Short: "Opcap",
+		Long:  "Long Opcap",
```

Comment:
is this the intended description ? or just a placeholder since its not used ?

Reply:
Oops. I'll make that better.